spec/services/create_guest_user_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+RSpec.describe CreateGuestUser, type: :controller do
+  describe '#call' do
```

Comment:
```suggestion
  describe '.call' do
```
Lembra da convenção de usar `.` para métodos estáticos